### PR TITLE
[JS]: Light and dark mode

### PR DESCRIPTION
**Pull request by ankit071105**
# Completd All the necessary changes and added Dark Mode Feature

### CODE REVIEW

**Comment by arpansaha13** [comment]
Please do not create a new PR after every review.
When you push a new commit to your branch they will automatically appear in this PR.
There is no need to create new PRs after every change.

File: index.html
Comment (sadanandpai) [inline index.html:1]: is there a need for changing the file?

File: .DS_Store
Comment (sadanandpai) [inline .DS_Store:1]: why is this needed

File: vue/index.html
Comment (sadanandpai) [inline vue/index.html:1]: is there a need to change the file?

File: native/index.js
Comment (sadanandpai) [inline native/index.js:1]: is there a need to change the file?

File: native/index.html [hunk 1]
```diff
@@ -40,19 +40,31 @@
...
+        <label for="switch" style="background-color: black; color: #080a0c;"></label>
+  </label>
+  <script>
+    const $html = document.querySelector("html");
+    const $checkbox = document.querySelector("#switch");
+    
+    $checkbox.addEventListener("change", function() {
+        $html.classList.toggle("light-mode");
+    });
+    </script>
```
Comment (sadanandpai) [inline native/index.html:67]: please dont keep script in hmtl

File: native/index.html [hunk 2]
```diff
@@ -40,19 +40,31 @@
         }
       });
     </script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css">
+
```
Comment (sadanandpai) [inline native/index.html:44]: can we not do it without font awesome

**Review by sadanandpai — CHANGES_REQUESTED** [review]

**Comment by ankit071105** [comment]
> # Completd All the necessary changes and added Dark Mode Feature

changes done as per the requested and made a new separate javascript file

**Review by sadanandpai — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -28,6 +31,10 @@
         window.location =
           "https://sadanandpai.github.io/frontend-mini-challenges/react/dist/";
       }
+
```
[the remark this answers is not in the document]
Comment (arpansaha13) [reply index.html:34]: I am not sure how that happened 🤔

File: native/index.html [hunk 4]
```diff
@@ -72,8 +76,15 @@ <h3 class="challenge-title"></h3>
             <img class="developer-img" src="" alt="" />
             <span class="developer-name"></span>
           </div>
+
```
[the remark this answers is not in the document]
Comment (arpansaha13) [reply native/index.html]: Oh this one is a habit of mine 🥲

**Pull request merged**